### ISSUE TITLE
Make it possible for contributors to start their own branches!

### ISSUE THREAD

**Issue by rduerr** [POLDER-Crew/Best-Practices-schema.org#8]
As seen below - I do not have access to create/delete, etc. branches....
![image](https://user-images.githubusercontent.com/1544892/98314968-58c72880-1f94-11eb-988d-cc8b17b1dc61.png)

**Comment by rduerr** [comment]
I also don't have the rights to assignee folks (including me), create labels, projects, milestones, etc.

**Comment by southern-ocean-observing-system** [comment]
You should definitely have all of those things!

**Comment by southern-ocean-observing-system** [comment]
Are you working in your own fork? Amos and I are looking through the repo now and we think you ought to be able to create pull requests against the master branch.

**Comment by mbjones** [comment]
Of course, anybody can create a fork and their own branches in their fork, and then create a pull request to ask for the maintainers to merge in the proposed PR changes to the upstream repository. Which I don't think is what @rduerr is asking about (am I right, Ruth?).  For me, the question here is who are those maintainers? Is there a list somewhere? Maybe it's the POLDER chairs? It might be helpful if the contribution process was outlined in a CONTRIBUTING.md document (like we did for [SOSO](https://github.com/ESIPFed/science-on-schema.org/blob/master/CONTRIBUTING.md)) so everyone knows what is expected, and for those new to PRs and this way of working, they might use it as a guide. I wrote the SOSO one and it is a bit too technical, but I think it's a step in the right direction.

My personal opinion is that it is useful to have more people with write and triage privileges on the upstream repository, as it is hard to build momentum if only one or two people can meaningfully help manage the repo. There's a bootstrapping process to contributions that requires multiple people to invest in labeling, milestone creation, issue management, mergining PRs, etc. I think there needs to be apparent, visible activity on a regular basis, which is hard for 1 or 2 volunteers by themselves. Even writing and responding to issues like this is an investment in community engagement.

**Comment by ahayes** [comment]
Hey @mbjones . I did have a look at your contributing doc and I think at the time (oh so long ago) it seemed a bit overkill for our little... crew. So I just put some basic info in the readme, including a link to some github docs on how to create a pull request, etc.

We have an email thread going where we had some discussion about the process for approving pull requests and I think there may be some consensus that we need more than one reviewer to approve a pull request to master. I have found button in github that let me set that. It is helpful because it hides/disables buttons until they are ok to press based on whatever the agreed approach is. We have at least one person in here who readily admits to pressing buttons without knowing what they do and that could help a bit. :)

I was worried about branch clutter but I have found an option that can be set to auto-delete branches after pull requests are accepted, so that could help. Otherwise everyone has to know/remember to check the box when accepting. So this could be a good compromise.

**Comment by rduerr** [comment]
@pbricher No I didn't create my own fork as I wanted to sort of use something similar to the SOSO process which involves creating a branch with your name, the issue # it is responding to, and a short title stating what it is about. 

I ended up creating a new repo under POLDER-Crew and put my stuff there...  I'll be happy to move it over, once the permissions are sorted out.  I note that I logged out and back in and still can't create a branch or do any of those other things that I am used to being able to do.  So you don't have my setting right quite yet.
